test/StripeResource.spec.js

```diff
@@ -49,11 +49,12 @@ describe('StripeResource', () => {
     });
 
     describe('_request', () => {
-      it('encodes the query string in GET requests', (done) => {
+      it('encodes the body in GET requests', (done) => {
```

Comment:
Fun fact, stripe-node always sends parameters in request bodies regardless of the HTTP method. The only exception was `retrieveUpcoming()` because it had a custom implementation.

Ideally, the library should send parameters in query strings for GET and DELETE requests and in request bodies for POST requests, but it doesn't look like a simple fix and the backend doesn't care.

Reply:
😮